base/index.mustache

```diff
@@ -4,28 +4,32 @@ const fs = require('fs');
 
 require('required_env')(fs.readdirSync('./env').reduce((r, file) => {
   return r.concat(require('./env/'+file));
-}, []))
+}, []));
 
-const restify = require('restify');
+const cors = require('cors');
+const app = require('express')();
+const bodyParser = require('body-parser');
+const server = require('http').createServer(app);
+const io = require('socket.io')(server);
 
-const app = restify.createServer();
-const io = require('socket.io').listen(app.server);
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+app.use(cors());
 
-app.use(restify.bodyParser());
-app.use(restify.queryParser());
-
-fs.readdirSync('./middleware').map(name => {
-  require('./middleware/'+name)(app);
-});
```

Comment:
Yeah, I removed as this as after the cors and jwt middlewares were removed and now simply done with `app.use`, I figure if I we - as authors - are adding anymore it'd be the same way; either application level middleware, or routing level middleware.  Because there are no middleware generators, then there is no need to read and load them dynamically.

If a future redbeard user wants to add middleware, sure, they can add a separate folder for it if they so wish.